src/App.js

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import './App.css';
-import SearchBar from './Components/SearchBar';
+import Home from './Components/SearchBar';
+import Repo from './Components/Repo';
 
 function App() {
+  <BrowserRouter>
+    <Switch>
+      <Route path="/" exact component={Home} />
+      <Route path="/repo/:author/:name" component={Repo} />
+    </Switch>
+  </BrowserRouter>;
+
   return (
     <div className="App">
-      <h1> Search Bar </h1>
-      <SearchBar />
+      <Home />
     </div>
```

Comment:
In your App file, Router is not returned - you return Home page. Nothing happens with the router. :) 
App.js should look like:
```
function App() {
  return (
    <BrowserRouter>
      <Switch>
        <Route exact path={PATHS.HOME} component={Home} />
        <Route path={PATHS.REPO} component={Repo} />
      </Switch>
    </BrowserRouter>
  );
}
`